Core/src/org/sleuthkit/autopsy/contentviewers/AnnotationsContentViewer.java

```diff
@@ -0,0 +1,433 @@
+                    }
+                }
+                
+                if (commentDataFound == false) {
+                    addMessage(html, "There is no comment data for the selected content in the central repository.");
+                }
+            } catch (EamDbException | TskCoreException ex) {
+                logger.log(Level.SEVERE, "Error connecting to the central repository database.", ex); // NON-NLS
+            }
+            endSection(html);
+        }
+    }
+
+    /**
+     * Set the text of the text panel.
+     * 
+     * @param text The text to set to the text panel.
+     */
+    private void setText(String text) {
+        jTextPane1.setText("<html><body>" + text + "</body></html>"); //NON-NLS
+    }
+    
+    /**
+     * Start a new data section.
+     * 
+     * @param html        The HTML text to add the section to.
+     * @param sectionName The name of the section.
+     */
+    private void startSection(StringBuilder html, String sectionName) {
+        html.append("<p style=\"font-size:14px;font-weight:bold;\">")
+                .append(sectionName)
+                .append("</p><br>"); //NON-NLS
+    }
+    
+    /**
+     * Add a message.
+     * 
+     * @param html    The HTML text to add the message to.
+     * @param message The message text.
+     */
+    private void addMessage(StringBuilder html, String message) {
+        html.append("<p style=\"font-style:italic;\">")
+                .append(message)
+                .append("</p><br>"); //NON-NLS
+    }
+    
+    /**
+     * Add a data table containing information about a tag.
+     * 
+     * @param html The HTML text to add the table to.
+     * @param tag  The tag whose information will be used to populate the table.
+     */
+    private void addTagEntry(StringBuilder html, Tag tag) {
+        startTable(html);
+        addRow(html, "Tag:", tag.getName().getDisplayName());
+        addRow(html, "Tag User:", tag.getUserName());
+        addRow(html, "Comment:", convertLineBreaksToHtml(tag.getComment()));
+        endTable(html);
+    }
+    
+    /**
+     * Add a data table containing information about a correlation attribute
+     * instance in the central repository.
+     * 
+     * @param html              The HTML text to add the table to.
+     * @param attributeInstance The attribute instance whose information will be
+     *                          used to populate the table.
+     * @param correlationType   The correlation data type.
+     */
+    private void addCentralRepositoryEntry(StringBuilder html, CorrelationAttributeInstance attributeInstance) {
+        startTable(html);
+        addRow(html, "Case:", attributeInstance.getCorrelationCase().getDisplayName());
+        addRow(html, "Type:", attributeInstance.getCorrelationType().getDisplayName());
+        addRow(html, "Comment:", convertLineBreaksToHtml(attributeInstance.getComment()));
+        addRow(html, "Path:", attributeInstance.getFilePath());
```

Comment:
Text visible to the user such as these labels and section headings should probably be defined as messages for localization purposes.